AdblockPlusSafari/Base.lproj/Localizable.strings

```diff
@@ -0,0 +1,20 @@
+/* 
```

Comment:
Yes we need it. Xcode project localization is scattered through several different places/files. It's all enumerated in XLIFF export. Localizable.strings is a resource for NSLocalizedString native calls and there is really just 4 of them in ABP. There should/will be more at expense of the Storyboard localization, which is Apple's preferred declarative UI vehicle, but mind boggingly unusable localization-wise. Meanwhile i will fix the header.